### ISSUE TITLE
Seeking a clear statement on library activity/staleness

### ISSUE THREAD

**Issue by Xerillio** [microsoft/azure-devops-node-api#568]
I've seen several issues being marked as stale and even closed automatically by the GitHub bot while the issue is awaiting feedback/resolution from maintainers. I know this is OSS and there might be higher priorities taking up the time, which is all fair - so this is no complaint. However, would it be possible to make a clear statement - preferrably in the README - about the intentions of this library and how users are expected to use it?

My main concern at the moment is that, I've seen more of these issues about functionality from the Azure DevOps API that are not implemented in this library and usually the repsonse is that this client is generated from server-side controllers. For that reason it seems difficult to contribute by e.g. implementing these missing parts. So I'm left wondering if this library is considered more of a quick boost to get started using the API rather than a fully-fledged client meant to support the entire API (now or in the future) and if I'm thus expected to implement the missing pieces in my own code-base.

Again, no hard feelings and thanks for making this library available. I'm just looking for a clear message.

**Comment by SvenStaehs** [comment]
indeed, I would also wonder whether this is service/cloud-only, like so many other things seem to be.
I just spent quite a while trying to find out why I can't get a connection going. Turns out during connection (in `vsoClient.getVersioningData()`) it tries to access an "_apis/Location" endpoint, which I can't actually find described anywhere...


If this module is meant for cloud only, I would like to see this statement in Readme.
If it's supposed to support ADO Server as well, I'll need to know that as well so I can confidently write a bug.
